### PR TITLE
feat(jenkins): Support skip comments

### LINKED ISSUE #6979: Jenkins manager to support skipping upgrade of certain plugins
**What would you like Renovate to be able to do?**

Currently, Jenkins manager finds all plugins in a file.
Sometimes one wants to pin a version so Renovate does not try to upgrade it, i.e. with a special comment:
```
plugin-name:7.8.9 # [renovate-ignore] Renovate will not upgrade this
```

**Did you already have any implementation ideas?**

Yes, the implementation PR is coming shortly.

**Are there any workarounds or alternative ideas you've tried to avoid needing this feature?**

None.

**Is this a feature you'd be interested in implementing yourself?**

Yes.

### PR DESCRIPTION
**Pull request by maxdanilov**
Closes #6979

### CODE REVIEW

**Comment by maxdanilov** [comment]
I believe there's a problem on renovate/renovate:master where `yarn lint` is failing, hence this build also has the lining job failing.

File: lib/manager/common.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ export interface ExtractConfig extends ManagerConfig {
   yarnrc?: string;
   skipInstalls?: boolean;
   versioning?: string;
+  ignoreComments?: string[];
```
Comment (rarkins) [inline lib/manager/common.ts:28]: I don't see any immediate value in passing this "up".

**Review by rarkins — CHANGES_REQUESTED** [review]
I don't think we want to pass such comments or logic outside the jenkins manager itself. Instead, we can simply set dep.skipReason = ignored.

Please check out https://github.com/renovatebot/renovate/search?q=isskipcomment&unscoped_q=isskipcomment and see if you can align with that.

**Comment by viceice** [comment]
@maxdanilov Why not simply use a package rule to disable / ignore specific dependencies?

**Comment by maxdanilov** [comment]
@viceice because regardless of whether one uses Renovate or not, they should be able to somehow mark the package version is pinned, and be able to also do it in the code itself.
I actually like the proposal of @rarkins to use a `skipComment`, will implement it when I have time.

**Comment by maxdanilov** [comment]
@rarkins I've implemented your suggestion for using `isSkipComment` instead - it looks much better now indeed!

**Pull request merged**